modules/helfi_node_landing_page/config/install/metatag.metatag_defaults.node__landing_page.yml

```diff
@@ -5,4 +5,3 @@ id: node__landing_page
 label: 'Content: Landing page'
 tags:
   description: '[node:field_hero:entity:field_hero_desc]'
-
```

Comment:
This config file is probably not related to this PR

Reply:
Not related. drush config export decided that this is important